.github/workflows/conformance-k8s-kind.yaml

```diff
@@ -0,0 +1,252 @@
+  push:
+    branches:
+      - master
+      - ft/master/**
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+
+permissions: read-all
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.event.after }}
+  cancel-in-progress: true
+
+env:
+  kind_version: v0.17.0
+  cluster_name: cilium-testing
+  # renovate: datasource=github-releases depName=cilium/cilium-cli
+  cilium_cli_version: v0.13.0
+  k8s_version: v1.26.0
+
+jobs:
+  kubernetes-e2e:
+    runs-on: ubuntu-latest
+    timeout-minutes: 45
+    strategy:
+      fail-fast: false
+      matrix:
+        # TODO add "dual" and "ipv6", "ipv6" fails to install cilium
+        ipFamily: ["ipv4"]
+    env:
+      IP_FAMILY: ${{ matrix.ipFamily }}
+
+    steps:
+      - name: Checkout master branch to access local actions
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+        with:
+          ref: ${{ github.event.repository.default_branch }}
+          persist-credentials: false
+      - name: Set Environment Variables
+        uses: ./.github/actions/set-env-variables
+
+      - name: Enable ipv4 and ipv6 forwarding
+        run: |
+          sudo sysctl -w net.ipv6.conf.all.forwarding=1
+          sudo sysctl -w net.ipv4.ip_forward=1
+
+      - name: Set up environment (download Kubernetes dependencies)
+        run: |
+          TMP_DIR=$(mktemp -d)
+          # Test binaries
+          curl -L https://dl.k8s.io/${{ env.k8s_version }}/kubernetes-test-linux-amd64.tar.gz -o ${TMP_DIR}/kubernetes-test-linux-amd64.tar.gz
+          tar xvzf ${TMP_DIR}/kubernetes-test-linux-amd64.tar.gz \
+            --directory ${TMP_DIR} \
+            --strip-components=3 kubernetes/test/bin/ginkgo kubernetes/test/bin/e2e.test
+          # kubectl
+          curl -L https://dl.k8s.io/${{ env.k8s_version }}/bin/linux/amd64/kubectl -o ${TMP_DIR}/kubectl
+          # kind
+          curl -Lo ${TMP_DIR}/kind https://kind.sigs.k8s.io/dl/${{ env.kind_version }}/kind-linux-amd64
+          # Install
+          sudo cp ${TMP_DIR}/ginkgo /usr/local/bin/ginkgo
+          sudo cp ${TMP_DIR}/e2e.test /usr/local/bin/e2e.test
+          sudo cp ${TMP_DIR}/kubectl /usr/local/bin/kubectl
+          sudo cp ${TMP_DIR}/kind /usr/local/bin/kind
+          sudo chmod +x /usr/local/bin/*
+          sudo rm -rf ${TMP_DIR}
+
+      - name: Create multi node cluster
+        run: |
+          cat <<EOF | /usr/local/bin/kind create cluster \
+            --name ${{ env.cluster_name}}           \
+            --image kindest/node:${{ env.k8s_version }}  \
+            -v7 --wait 1m --retain --config=-
+          kind: Cluster
+          apiVersion: kind.x-k8s.io/v1alpha4
+          networking:
+            ipFamily: ${IP_FAMILY}
+            kubeProxyMode: "none"
+          nodes:
+          - role: control-plane
+          - role: worker
+          - role: worker
+          EOF
+
+      - name: Workaround CoreDNS for IPv6 airgapped
```

Comment:
this step seems only required once we enable ipv6 testing.

Reply:
yeah, check that the step is skipped in that case
```
        if: ${{ matrix.ipFamily == 'ipv6' }}
```

Reply:
thanks